sale_order_line_delivery_state/views/sale_order_views.xml

```diff
@@ -18,14 +18,14 @@
                         type="object"
                         string="Force delivery done"
                         states="done"
-                        attrs="{'invisible': ['|', ('force_delivery_state', '=', True), ('delivery_state', '=', 'done')]}"
+                        invisible="['|', ('force_delivery_state', '=', True), ('delivery_state', '=', 'done')]"
```

Comment:
Does this work? I thought a Python expression was expected here, like `force_delivery_state or delivery_state == 'done'`